client/app/src/services/toast.service.js

```diff
@@ -0,0 +1,97 @@
+(function() {
+  'use strict';
+
+  var fs = require('fs');
+
+  angular.module('arkclient.services')
+    .service('toastService', ['configService', '$mdToast', 'gettextCatalog', ToastService]);
+
+  /**
+   * ToastService
+   * @constructor
+   */
+  function ToastService(configService, $mdToast, gettextCatalog) {
```

Comment:
getTextCatalog*

Reply:
This is passed in. I agree, I'd use camelcase normally but wanted to keep it consistent with the value on line 6/7 (as well as other places like `account.controller.js`):

```
angular.module('arkclient.services')
  .service('toastService', ['configService', '$mdToast', 'gettextCatalog', ToastService]);
```